src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/AutoCompleteResourceV2Feed.cs

```diff
@@ -45,7 +44,8 @@ public AutoCompleteResourceV2Feed(HttpSourceResource httpSourceResource, string
         {
             var apiEndpointUri = new UriBuilder(new Uri(_baseUri, @"package-ids"))
             {
-                Query = $"partialId={packageIdPrefix}&includePrerelease={includePrerelease.ToString()}"
+                Query = $"partialId={packageIdPrefix}&includePrerelease={includePrerelease.ToString()}" +
```

Comment:
Nope, not needed. String interpolation does this by default, as you mentioned. It's pre-existing code but you're right, I should fix it up.